doc/content/draft-template/_index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Draft Template"
+description: "This page is a draft template"
+weight: 
+draft: true
+---
+
+Guidelines here are taken from [DEVELOPMENT.md](https://github.com/TheThingsNetwork/lorawan-stack/blob/master/DEVELOPMENT.md), which you should read. This template exists as a copy and paste starting point for new documentation. Remove the `draft`key in the Front Matter to generate the page (otherwise hugo will skip it).
+
+A documentation page starts with an introduction, and then the first heading. The first paragraph of the introduction is typically a summary of the page. Use a <!--more--> to indicate where the summary ends.
+
+<!--more-->
+
+## Requirements
+
+Use a requirements subheading to list requirements/prerequisites.
+
+1. Requirement 1
+2. Requirement 2
+
+## Sections
+
+When possible, divide long documents into separate files, each with its own folder and `_index.md`. Use the `weight`tag in the [Front Matter](https://gohugo.io/content-management/front-matter/) to manually sort sections if necessary. If not, they will be sorted alphabetically.
+
+## Subheadings
+
+Since the title is a h1, everything in the content is at least h2 (##). Use subheadings to break up sections and provide a high level overview of steps the user should follow.
+
+## Notes
+
+Use the `>` markdown code, and include the word note to distinguish it from a code block.
+
+>Note: if you need help with any CLI command, use the `--help` flag to get a list of subcommands, flags and their description and aliases.
+
+## Images
+
+Screenshots should be inserted using the figure shortcode:
+
+{{< figure src="screenshot.png" alt="I am a screenshot" >}}
```

Comment:
Add instructions on how to take screenshots

I'm not sure what the instructions were exactly, but I thought it was using Chromium (Chrome, Brave, etc) developer tools, set display to "Laptop with HiDPI screen", then use the colons button for "Capture full size screenshot":

<img width="1303" alt="Screenshot 2020-06-03 at 08 43 35" src="https://user-images.githubusercontent.com/13334001/83604419-6a844a00-a576-11ea-8bcd-e2688632c880.png">